webClient/src/app/app.component.html

```diff
@@ -18,7 +18,7 @@
       <zlux-button class="iframe-button shadowed" type="button" (click)="sendAppRequest()"
       i18n="send request to application|Send a request to another application on the desktop@@send-app-request">Send App Request</zlux-button>
       <zlux-button class="iframe-button shadowed" type="button" (click)="getDefaultsFromServer()"
-      i18n="get from server@@get-from-server">Get from Server</zlux-button>
+      i18n="get from server@@plugin-loading-failed-message">Get from Server</zlux-button>
```

Comment:
Though you replaced the string everywhere so it probably works, Get From Server does not mean plugin loading failed message.

Reply:
fixed